api/voice-react-native-sdk.api.md

```diff
@@ -1172,16 +1139,16 @@ export namespace Voice {
         export type Registered = () => void;
         export type Unregistered = () => void;
     }
-    export type RunPreflightTestOptions = Partial<{
-        [Constants.CallOptionsKeyIceServers]: CallOptions.IceServer[];
-        [Constants.CallOptionsKeyIceTransportPolicy]: CallOptions.IceTransportPolicy;
-        [Constants.CallOptionsKeyPreferredAudioCodecs]: CallOptions.AudioCodec;
-    }>;
+    export interface PreflightOptions {
+        // Warning: (ae-forgotten-export) The symbol "CallOptions" needs to be exported by the entry point index.d.ts

```

Comment:
Do we need this here?

Reply:
I think we need to export the `CallOptions` member at some point yeah. This warning is autogenerated btw, the whole file is, since it's just the API rollup.